docs/kyma-environment-broker/08-06-orchestration-status.md

```diff
@@ -150,7 +147,7 @@ A successful call returns the list of all orchestrations:
    export OPERATION_ID={SCHEDULED_OPERATION_ID}
    ```
 
-2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](#details-orchestration) to fetch the details for given operation.
+2. Make a call to the Kyma Environment Broker with a proper **Authorization** [request header](03-10-orchestration.md) to fetch the details for given operation.
```

Comment:
I cannot see any request header details in the linked document. Did you mean `03-05-authorization.md`?